util/other.syms

```diff
@@ -476,6 +476,7 @@ SSL_CTX_set1_sigalgs_list               define
 SSL_CTX_set1_verify_cert_store          define
 SSL_CTX_set_current_cert                define
 SSL_CTX_set_ecdh_auto                   define
+SSL_CTX_set_dh_auto                     define
```

Comment:
Nit: are we trying to have some kind of alphabetical order in this file? :wink:

Reply:
I'm redefining alphabetical order :-)

Reply:
Hmmm, so we should expect a new locale then?  :wink:
(en_GB.matt)